Testscripts/Windows/LIS-INSTALL-SCENARIOS.ps1

```diff
@@ -491,6 +491,12 @@ Function Main {
         }
         #PROVISION VMS FOR LISA WILL ENABLE ROOT USER AND WILL MAKE ENABLE PASSWORDLESS AUTHENTICATION ACROSS ALL VMS IN SAME HOSTED SERVICE.
         Provision-VMsForLisa -allVMData $AllVMData -installPackagesOnRoleNames "none"
+        $sts = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username "root" -password $password `
+             -command "lsmod | grep hv_vmbus || echo 1" -ignoreLinuxExitCode:$true
```

Comment:
you don't have to echo 1, "lsmod | grep hv_vmbus" should be OK, then $sts would be empty if hv_vmbus is builtin